bool fr_rl_swinging
float32 x_velocity
float32 y_velocity
float32 theta_velocity
float32 fl_x
float32 fl_y
float32 fr_x
float32 fr_y
float32 rl_x
float32 rl_y
float32 rr_x
float32 rr_y
---
float32[] predictions